forms/chapter-organizer-form.md

```diff
@@ -0,0 +1,70 @@
+# PyLadies Chapter Organizer Registration Form
+
+Please fill in this form if you are:
+
+- Currently a PyLadies chapter organizer
+- Was a chapter organizer and wants to step down
+- A new organizer for an existing chapter.
+
+Please share this form with your co-organizers, as well as please remember
+to fill this form again with any changes in your chapter (i.e. someone
+new is joining the team or someone is stepping down).
+
+We may request you to fill this form annually, to keep our records up to date.
+
+## How will the data here be used?
+
+PyLadies [Admin-and-logistics team members](https://github.com/pyladies/project-admin-logisitics#team)
+will refer to this data if we need to get in touch
+with local chapter organizers, for example for verifying the chapter's status
+or for verifying password reset request.
+
+Other PyLadies members and organizers may want to contact a chapter directly
+for various purposes, for example: sharing conference discount codes,
+sharing job opportunities, request speakers from your community, et.
+
+## Who has access to this form?
+
+The information collected here will be made available to all PyLadies chapter
+organizers, as a Google Spreadsheet. The Google Spreadsheet can be viewed
+by anyone who logged in using the `{chapter}@pyladies.com` account.
+
+If you are a chapter organizer and would like access to this data,
+you can go to "(url tbd)" and log in using your chapter's `{chapter@pyladies.com}`
+email address.
+
+## Questions and concerns?
+
+Please bring this up in: <https://github.com/pyladies/info> and open an issue.
+
+Est. 2020
+
+# Form Questions
+
+- (Personal) Email address: (textbox)
+
+  Your own personal email address, which we can use to contact you
+  
+- PyLadies.com Email address: (textbox)
+
+  Your chapter's pyladies.com email address. Example: `vancouver@pyladies.com`
+
+- Your Name: (textbox)
+
+- Your pronouns: (options)
+  
+  - she/her
+  - he/his
+  - they/them
+  - ve/ver/vis
+  
+- Your status: (options)
```

Comment:
It's not clear to me how someone would use this form to remove themselves or others from being an organizer. 

Recommend we structure the form as follows:

- Basic minimum identifying information (email or whatever we are using for primary ID for Sheet)
- Conditional question: Your status: active organizer or inactive organizer
  - Active organizer leads to more questions to update/add PyLady to Sheet
  - Inactive organizer asks conditional question "Would you like to remove yourself as an organizer?"
    - If "Yes", remove + send email w/ results
   - If "No", take the questions to update Sheet, while updating status as "Inactive"

Reply:
@amcasari what other questions do you want to ask the organizer(s) if they are active? We are separately working on a membership form: https://github.com/pyladies/global-organizing/issues/54.

Reply:
> Inactive organizer asks conditional question "Would you like to remove yourself as an organizer?"

I think that if someone becomes inactive organizer, we should just remove them: remove from our records, and remove them from organizers slack. If they become active again, they can re-register and we can re-add them.

> Active organizer leads to more questions to update/add PyLady to Sheet

For the questions within this form, I think only the two questions relevant to active organizers are: pronoun and GitHub username. The Slack handle, we need it regardless so we can look them up on Slack and remove them from the organizers channel. I can make the "pronoun" and "GitHub username" as conditional questions. What do you think?

Reply:
:+1: to making the GitHub handle conditional. I, however, think we should as pronoun regardless.